README.md

```diff
@@ -36,12 +42,14 @@ let g:chat_gpt_max_tokens=2000
 let g:chat_gpt_model='gpt-4'
 let g:chat_gpt_session_mode=1
 let g:chat_gpt_temperature = 0.7
+let g:chat_gpt_lang = 'in Chinese, '
```

Comment:
i believe this should just be `Chinese`, since the additional `in` and `,` are in the template.